app/controllers/application_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ApplicationController < ActionController::Base
+  # before_action :authenticate_user!
+  before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
Please kindly remove unused code for it makes your code base look untidy. Please implement this change i all files where you have unused code.